README.md

```diff
@@ -256,6 +257,10 @@ Please see [the Docker installation
 documentation](https://docs.docker.com/installation/) for details on how to
 upgrade your Docker daemon.
 
+## Supported Node.js versions
+
+This project will support Node.js versions as still under active support as per the [Node.js release schedule](https://github.com/nodejs/Release).
```

Comment:
Maybe mention something about, that when the unsupported Dockerfiles are removed from this repo, the images still remain on the DockerHub?